app/views/state_file/questions/id_review/edit.html.erb

```diff
@@ -20,6 +21,12 @@
         <p><%=number_to_currency(current_intake.health_insurance_paid_amount || 0, precision: 2)%></p>
         <%= link_to t("general.edit"), StateFile::Questions::IdHealthInsurancePremiumController.to_path_helper(return_to_review: "y"), class: "button--small" %>
       </div>
+
+      <div class="spacing-below-5">
+        Idaho Grocery Credit
+        <%= current_intake.calculator.grocery_credit_amount %>
+        <%= link_to t("general.review_and_edit"), StateFile::Questions::IdGroceryCreditReviewController.to_path_helper(return_to_review: "y"), class: "button--small" %>
```

Comment:
it looks like we use `edit` for the other blocks here and "review and edit" for just this grocery credit -- looking at the figma it looks like we want "Review and Edit" for all the other buttons on this page, could we confirm with product?

Reply:
just double checked with product, and it should be Edit everywhere. Updated 👍